changelogs/2.11.6.md

```diff
@@ -0,0 +1,158 @@
+all the new features of the 2.x series.
+
+[upgrade]: https://www.tarantool.io/en/doc/latest/book/admin/upgrades/
+
+## Bugs fixed
+
+### Core
+
+* Fixed a bug when a server could crash if a client sent an IPROTO replication
+  request without waiting for pending requests to complete (gh-10155).
+* Fixed a bug when the `sequence:reset()` call result was not recovered
+  after the server restart (gh-9871).
+* Fixed several bugs in the numbers comparison and hashing in tuple keys. It
+  allowed to insert the same key multiple times into a unique index, and
+  sometimes wouldn't allow to find an existing key in an index. Could happen
+  when numbers were encoded in MessagePack suboptimally and when the `double`
+  field type was used (gh-9965).
+* Fixed the issue with hanging write operations forever triggered by heavy
+  write load and WAL writing failures on cascade rollback (gh-11081).
+
+### Memtx
+
+* Fixed a crash when memtx MVCC tried to abort an already committed
+  non-memtx transaction if it used a system space view or tried to
+  perform a DDL operation (gh-10715).
+* Fixed a bug when Tarantool with memtx MVCC enabled was aborted on
+  workload with many `index:get()` operations reading nothing (gh-11022).
+
+### Vinyl
+
+* Fixed a bug when a deleted tuple wasn't purged from a secondary index data
+  stored on disk in case it was updated more than once in the same transaction.
+  The bug couldn't result in inconsistent query results, but it could lead to
+  performance degradation and increased disk usage (gh-10820, gh-10822).
+* Fixed a bug when joining a new replica to a master instance that experiences
+  a heavy write load would severely degrade the master instance performance.
+  The fix should also speed up long-running scan requests (gh-10846).
+* Fixed a bug when a tuple could disappear from a multikey index in case it
+  replaced a tuple with duplicate multikey array entries created in the same
+  transaction. With the enabled `defer_deletes` space option, the bug could
+  also trigger a crash (gh-10869, gh-10870).
+* Fixed a bug when the tuple cache was not properly invalidated in case
+  a WAL write error occurred while committing a `space.delete()` operation.
+  The bug could lead to a crash or an invalid read query result (gh-10879).
+* Fixed a bug when a deleted secondary index key wasn't purged on compaction
+  of a space with the `defer_deletes` option enabled (gh-10895).
+* Fixed a bug in the tuple cache when a transaction operating in a read view
+  could skip a tuple deleted after the read view creation (gh-11079).
+* Fixed a bug in the tuple cache when a tuple could become inaccessible via
+  a secondary index after a transaction rollback caused by a WAL write error
+  (gh-11140).
+
+### Replication
+
+* RAFT requests are now sent with 0 (not local) GROUP_ID (gh-10727).
+* Fixed a bug when the Raft state wasn't sent during the `META_JOIN` stage,
```

Comment:
```suggestion
* Fixed a bug when the RAFT state wasn't sent during the `META_JOIN` stage,
```

Reply:
Fixed.

Reply:
Sent for `3.2.2` in #11197. Can't send it for 3.3.2 and 3.4.0, because the change is already released in 3.3.0.